tool/cli.js

```diff
@@ -891,6 +893,61 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
```

Comment:
OK, it occurs to me now that, based on my other suggestions, that this could conceivably expand to cover some non-frontmatter stuff. How about just "inventory" as the command name?

```suggestion
    "inventory",
```